modules/gapi/include/opencv2/gapi/render.hpp

```diff
@@ -0,0 +1,53 @@
+#ifndef OPENCV_GAPI_RENDER_HPP
+#define OPENCV_GAPI_RENDER_HPP
+
+#include <string>
+#include <vector>
+
+#include <opencv2/gapi/opencv_includes.hpp>
+#include <opencv2/gapi/util/variant.hpp>
+#include <opencv2/gapi/own/exports.hpp>
+#include <opencv2/gapi/own/scalar.hpp>
+
+namespace cv
+{
+namespace gapi
+{
+
+struct TextEvent
+{
+    std::string text;
+    int x;
+    int y;
+    int ff;
+    double fs;
+    cv::gapi::own::Scalar color;
+    int thick;
+    int lt;
+    bool bottom_left_origin;
+};
+
+struct RectEvent
+{
+    int x;
+    int y;
+    int widht;
+    int height;
+    cv::gapi::own::Scalar color;
+    int thick;
+    int lt;
+    int shift;
+};
+
+using DrawEvent  = util::variant<TextEvent, RectEvent>;
```

Comment:
`cv::gapi::draw::Prim`[itive]. Prim is ok

Reply:
Fixed